### DISCUSSION TITLE
参考文献中去掉//的方式

### DISCUSSION THREAD

**Discussion by BIOINSu** [mengchaoheng/SCUT_thesis#22] [General]
使用如下方式即可在参考文献中删除//：
\usepackage[backend=biber,style=gb7714-2015,gbalign=gb7714-2015,gbpunctin=false,gbpub=false,gbnamefmt = lowercase]{biblatex}
\DefineBibliographyStrings{english}{in={}}
另外，感谢大佬的模板，祝大家顺利毕业~

**Comment by mengchaoheng** [comment]
太感谢啦，我今天也有更新，根据博士论文送审结果进行了微调。包括：

新增模版第四章，还有公式段前短后间距调整、表格与插图目录内容间距修改、定理环境斜体改为正体。

**Comment by mengchaoheng** [comment]
> 使用如下方式即可在参考文献中删除//：
> 
> \usepackage[backend=biber,style=gb7714-2015,gbalign=gb7714-2015,gbpunctin=false,gbpub=false,gbnamefmt = lowercase]{biblatex}
> 
> \DefineBibliographyStrings{english}{in={}}
> 
> 另外，感谢大佬的模板，祝大家顺利毕业~

可以根据我刚刚的最新更新提交终稿，最好再仔细检查一下。

**Comment by BIOINSu** [comment]
收到🙋‍感谢大佬的持续贡献~因为看到学校发的规范没有//所以去搜了一下解决方案分享给想要的同学。我直接close掉这个issue啦，再次感谢大佬哈哈

**Reply by mengchaoheng** [reply]
> 收到🙋‍感谢大佬的持续贡献~因为看到学校发的规范没有//所以去搜了一下解决方案分享给想要的同学。我直接close掉这个issue啦，再次感谢大佬哈哈

我把它改为discussion了，再打扰一下，请问这个//的问题是审稿意见指出格式不对吗？我在考虑要不要添加这一项修改。

**Reply by BIOINSu** [reply]
哦这个不是审稿人提出的意见。
因为我们发的论文撰写规范要求符合GB7714-87的要求，所以没有加上//。而大佬你的模板应该是GB7714-2015的，这里是有//的，所以应该是符合规范的，只是规范不同而已。

**Comment by mengchaoheng** [comment]
package_Documentaion里面的biblatex-gb7714-2015.pdf的13页说明了这个符号的控制及含义：
```
为控制 inbook,incollection,inproceedings 中析出来源文献前的//符号而增加的选项。

-  gbpunctin=true，根据 GB/T 7714-2015 要求输出//。
- gbpunctin=false，则输出默认的本地字符串，在英语中是in:，若要完全去掉该
符号则可以在导言区增加命令 \DefineBibliographyStrings{english}{in={}}， \DefineBibliographyStrings{english}{incn={}}。之所以用加 cn 的本地化字 符串是为了适应某些样式对中英文文献的区别设置。
```

**Comment by mengchaoheng** [comment]
好像是需要有//的。知乎有一篇文章 [参考文献著录格式（国标GB／T7714-2015）](https://zhuanlan.zhihu.com/p/355312827)讲的很好，配合本库所给国标文档可以搞清楚。这里主要留给大家参考。
